### ISSUE TITLE
gmaven-plugin has been discontinued

### ISSUE THREAD

**Issue by wholegroup** [jcabi/jcabi-parent#29]
gmaven-plugin has been discontinued and there is a dependency incompatibility between groovy-all 2.3.3 and gmaven-plugin that uses groovy 2.0 (see providerSelection = 2.0). 
It leads to errors with Spock Framework.
gmaven-plugin should be replaced by gmavenplus-plugin.

```
				<plugin>
					<groupId>org.codehaus.gmavenplus</groupId>
					<artifactId>gmavenplus-plugin</artifactId>
					<version>1.5</version>
				</plugin>
```

**Comment by wholegroup** [comment]
@yegor256 please confirm

**Comment by yegor256** [comment]
@wholegroup definitely a bug, thanks for reporting

**Comment by dmarkov** [comment]
@wholegroup we will find someone to do this task, soon

**Comment by dmarkov** [comment]
@wholegroup thanks a lot for reporting, **15 mins** added to your acc, pmt ID `55852767`

**Comment by dmarkov** [comment]
@yamstranger please go ahead with this task, it's yours

**Comment by YamStranger** [comment]
@dmarkov assign to someone else. Sorry.  It seems I was in @away mode when this task was assigned. @yegor256 can you check it please? Thanks.

**Comment by dmarkov** [comment]
> @dmarkov assign to someone else. Sorry.  It seems I was in @away mode when this task was assigned. @yegor256 can you check it please? Thanks.

@yamstranger I deducted 30 points from your rating

**Comment by dmarkov** [comment]
> @dmarkov assign to someone else. Sorry.  It seems I was in @away mode when this task was assigned. @yegor256 can you check it please? Thanks.

@yamstranger all right, we'll find someone else for this task

**Comment by dmarkov** [comment]
@suseika please go ahead, it's your task now, keep [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) in mind, and don't hesitate to ask any technical questions you may have. The budget of this task is **30 mins**. This is exactly how much time will be compensated, when the task is completed. Read about our [Definition of Done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by dmarkov** [comment]
@rultor @wholegroup please take a look, rultor closed it automatically

**Comment by wholegroup** [comment]
@Suseika Thanks!

**Comment by dmarkov** [comment]
@suseika I haven't seen any msgs from you, won't pay (correct me if I'm wrong)

**Comment by Suseika** [comment]
@dmarkov What messages are you talking about?

**Comment by yegor256** [comment]
@dmarkov please add extra 30 mins to @Suseika

**Comment by Suseika** [comment]
@dmarkov Yegor explained what was my fault, sorry, won't happen again.

This was closed by #32

**Comment by dmarkov** [comment]
> @dmarkov please add extra 30 mins to @Suseika 

@yegor256 I've paid 30 mins to @suseika in transaction `AP-0XK26774DK778222J`
